Outreachy Dec 2021 to Mar 2022.md

```diff
@@ -0,0 +1,36 @@
+
+### Reaching and defining pilots plan with NGOs
+
+* For reaching and drafting pilot plans with NGOs, we recommend that you start finding and listing a few NGOs related to children education that you could potentially work with to use Oppia lessons. You will need to contact and meet with the NGOs in order to introduce Oppia and talk about the NGO activities. 
+* In this meeting, try to understand their pains and needs to better find a way for us to work together in a mutual beneficial partnership. You may make a copy and personalize this [presentation slides ](https://docs.google.com/presentation/d/1kWXni2omdpOZjUId712odYGSxfEziHg0wu81ChdvHAE/edit?usp=sharing) to help you pitch Oppia to the partner.
+* After you have a good understanding of the NGO context, work with them to draft a pilot plan detailing how they can include Oppia in their activities. If you think you need help on how to prioritize which NGOs to reach and how elaborate a nice reach out message, you can email our mentors for more materials.
+
+### Reaching and defining pilots plan with NGOs
+* For this task, you will need to create a plan on how you would scale the lessons and provide them to a larger audience in your country. It's important for the plan to be supported by reliable research that can include: (a) a benchmark analysis of similar products in your region, (b) an analysis of your country's/region's demographics, and (c) an analysis of existing NGOs and potential partners. 
+* You can use any documents and research you find relevant to support your proposal. The format of the plan is completely up to you. 
+* If you need help with the background research for your plan, you can contact the mentors for further guidance and additional materials.
+
+## FAQ
+
+**Q: How can I contribute with the project?**
+
+A: After finishing the required tasks, add a contribution URL to your Outreachy portal with all the necessary details and links. (The Contribution URL for each task would be a link to a Google Drive where you have uploaded the document/spreadsheet for that specific task.) Make sure that the file access is public!
+
+**Q: How can I translate a lesson?**
+
+A: To translate a lesson, **first** fill out this contributor agreement to confirm that you’re happy to submit contributions under the Creative Commons license. **Secondly** read through [this document](https://docs.google.com/document/d/17jMFtfHVWtJYrzyGQUKdsRXgky7lWv76sGYLOxSbA5w/edit?usp=sharing)  with information on how the translation process works, and let me know if you have any questions.  After you've filled out the form and read the documentation, please contact the mentors by email for further guidance.
```

Comment:
There's no link for the contributor agreement

Reply:
```suggestion
A: To translate a lesson, **first** fill out this contributor agreement to confirm that you’re happy to submit contributions under the Creative Commons license. **Second,** read through [this document](https://docs.google.com/document/d/17jMFtfHVWtJYrzyGQUKdsRXgky7lWv76sGYLOxSbA5w/edit?usp=sharing)  with information on how the translation process works, and let me know if you have any questions.  After you've filled out the form and read the documentation, please contact the mentors by email for further guidance.
```

This is nit-picky, but I think it's easier to read when "first" and "second" match (vs "first" and secondly")

Reply:
Who is "me"? How should readers figure out who the mentors are what their email addresses are? (maybe mention that this info is available from Outreachy)